README.md

```diff
@@ -0,0 +1,33 @@
+## START SERVER
```

Comment:
- [ ] Your readme does not follow the [Microverse Readme Template](https://github.com/microverseinc/readme-template/blob/master/README.md).  Please, kindly implement it to your project so it looks professional.
- [ ] Please, don't forget to add detailed instructions about how to setup and run your app.
- [ ] Kindly add the admin credentials to your readme, or to the pull request so it is possible to add or remove developers.

That way, your app will be well documented 👍